script/intentfest/validate.py

```diff
@@ -174,7 +174,7 @@ def validate(value):
         vol.Required("language"): str,
         vol.Optional("responses"): {
             vol.Optional("intents"): {
-                str: {vol.Required("success"): [str]},
+                str: {vol.Required("success"): [{"templates": [str]}]},
```

Comment:
Add a comma after the list element to force a new line for readability.

```suggestion
                str: {vol.Required("success"): [{"templates": [str]},]},
```